apps/events-helsinki/src/domain/search/eventSearch/constants.tsx

```diff
@@ -262,19 +262,19 @@ export const THEATRE_COURSES_KEYWORDS = [
 ];
 
 // todo: replace with valid keyword ids
-export const MAPPED_COURSE_CATEGORIES: Record<string, string[]> = {
-  [COURSE_CATEGORIES.MOVIE]: MOVIES_AND_MEDIA_COURSES_KEYWORDS,
-  [COURSE_CATEGORIES.LANGUAGES]: LANGUAGES_COURSES_KEYWORDS,
-  [COURSE_CATEGORIES.LITERATURE]: LITERATURE_COURSES_KEYWORDS,
-  [COURSE_CATEGORIES.ARTS_AND_CULTURE]: ARTS_AND_CULTURE_COURSES_KEYWORDS,
-  [COURSE_CATEGORIES.VISUAL_ARTS]: VISUAL_ARTS_COURSES_KEYWORDS,
-  [COURSE_CATEGORIES.HANDICRAFTS]: HANDICRAFTS_COURSES_KEYWORDS,
-  [COURSE_CATEGORIES.SPORT]: SPORT_COURSES_KEYWORDS,
-  [COURSE_CATEGORIES.MUSIC]: MUSIC_COURSES_KEYWORDS,
-  [COURSE_CATEGORIES.GAMES]: GAMES_COURSES_KEYWORDS,
-  [COURSE_CATEGORIES.FOOD]: FOOD_COURSES_KEYWORDS,
-  [COURSE_CATEGORIES.DANCE]: DANCE_COURSES_KEYWORDS,
-  [COURSE_CATEGORIES.THEATRE]: THEATRE_COURSES_KEYWORDS,
+export const MAPPED_EVENT_CATEGORIES: Record<string, string[]> = {
```

Comment:
`Record< EVENT_CATEGORIES, string[]>` probably would give more type safety and nags if you forget to add some enum category to the object.